src/Web/Sfa.Das.Sas.Web/Views/Apprenticeship/Standard.cshtml

```diff
@@ -153,10 +153,10 @@
 @helper GetMoreInformationProperty(string link)
 {
     {
-        var details = "The Institute for Apprenticeships has more detailed information about this apprenticeship.";
+        var details = "The <a href='" + "https://www.instituteforapprenticeships.org/" + "' target='_blank'>Institute for Apprenticeships and Technical Education</a> has more detailed information about this apprenticeship.";
         if (link != null && link.Trim() != string.Empty)
         {
-            details = @"The Institute for Apprenticeships has more <a href='" + link + "' target='_blank'>detailed information</a> about this apprenticeship.";
+            details = @"The Institute for Apprenticeships and Technical Education has more <a href='" + link + "' target='_blank'>detailed information</a> about this apprenticeship.";
```

Comment:
Should the link to IFA not be on both of these options instead of just the one without the detailed info about the apprenticeship?

Reply:
No, we only link to the root of the ifa site if the specific standard url isn't pulled through

Reply:
👍